.changeset/deprecate-actionlistv1-promote-actionlistv2.md

```diff
@@ -0,0 +1,152 @@
+---
+'@primer/react': major
+---
+
+### ActionList
+
+ActionList now ships a composable API.
+
+See full list of props and examples: https://primer.style/react/ActionList
+
+<table>
+<tr>
+<th> Before </th> <th> After </th>
```

Comment:
I'm making the following change in the tech preview post, as I think it makes it more obvious.. what do you think?

```suggestion
<th> Before (v34)</th> <th> After (v35)</th>
```

Reply:
Done!